instrumented/integration/src/main/kotlin/com/datadog/android/sdk/integration/rum/ActivityTrackingPlaygroundActivity.kt

```diff
@@ -20,21 +20,19 @@ import com.datadog.android.sdk.utils.getTrackingConsent
 internal class ActivityTrackingPlaygroundActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        // use the activity view tracking strategy
-        val config = DatadogConfig.Builder(
-            RuntimeConfig.DD_TOKEN,
-            RuntimeConfig.INTEGRATION_TESTS_ENVIRONMENT,
-            RuntimeConfig.APP_ID
-        )
-            .useCustomLogsEndpoint(RuntimeConfig.logsEndpointUrl)
-            .useCustomTracesEndpoint(RuntimeConfig.tracesEndpointUrl)
-            .useCustomRumEndpoint(RuntimeConfig.rumEndpointUrl)
+        super.onCreate(savedInstanceState)
```

Comment:
if you move this here wouldn't this mean that we're going to lose some events ?

Reply:
Nope, we don't miss anything